SETUP/install_db.php

```diff
@@ -18,10 +18,10 @@
 
 mysqli_query(DPDatabase::get_connection(), "
     CREATE DATABASE IF NOT EXISTS $db_name
-") or die(mysqli_error(DPDatabase::get_connection()));
+") or die(DPDatabase::log_error());
 mysqli_query(DPDatabase::get_connection(), "
     USE $db_name
-") or die(mysqli_error(DPDatabase::get_connection()));
+") or die(DPDatabase::log_error());
```

Comment:
There's also the occurrence on line 49.

Reply:
Ah - actually I want to roll back my changes and leave the original. I anticipate that the user will want the raw SQL error when running this script on the command line. Thanks for catching this.